### PR TITLE
[redesign] Header and Page layout

### LINKED ISSUE #72: [redesign] Implement new header
I am breaking down the implementation of the [new design](https://github.com/decred/dcrtimegui/issues/67) into smaller issues so we don't need to tackle it all at once. 
This issue covers only the implementation of the new Header, and as expected, the "hash search" functionality as well.
Screen Shot 2019-07-29 at 06 13 30

### PR DESCRIPTION
**Pull request by thi4go**
This PR introduces `pi-ui` layout to the pages, as well as the new Header design with the search bar. Functionality of the result page, searching a digest, will be refactored on the PR referent to the result page.

closes #72 
part of #85

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 import { BrowserRouter as Router } from "react-router-dom";
+import { defaultLightTheme, useTheme } from "pi-ui";
 import Routes from "./Routes";
 
 const App = () => {
+  useTheme(defaultLightTheme);
   return (
     <Router>
-      <Routes />
+      <Suspense fallback={<div />}>
```
Comment (tiagoalvesdulce) [inline src/App.js]: We need a loader component here. Pi-ui has a Spinner component.

File: src/components/layout/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./layout.module.css";
+
+const Footer = () => (
+  <div className={styles.footer}>
+    <div className={styles.textWrapper}>
+      <span>Decred developers | 2016 - 2019</span>
```
Comment (tiagoalvesdulce) [inline src/components/layout/Footer.jsx]: This "2019" shouldn't be hardcoded. Please use something like: `const year = new Date().getFullYear();`

File: src/App.js (hunk 1 above)
Comment (thi4go) [reply src/App.js]: Forgot to add it here, sorry about that

**Comment by thi4go** [comment]
Pushed the fixes @tiagoalvesdulce :+1:

**Pull request merged**